bundles/org.eclipse.platform.doc.isv/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.platform.doc.isv; singleton:=true
-Bundle-Version: 4.27.0.qualifier
+Bundle-Version: 4.27.1.qualifier
```

Comment:
corresponding pom update required

Reply:
https://github.com/eclipse-platform/eclipse.platform.common/blob/8c24111566f2be2018deaec632fdb29c05683c2d/bundles/org.eclipse.platform.doc.isv/pom.xml#L20-L23

it does not have a version section; looks like it inherits the parent's one?

Reply:
line17 has the version